src/tests/system/requirements.txt

```diff
@@ -5,4 +5,4 @@ git+https://github.com/next-actions/pytest-mh
 git+https://github.com/next-actions/pytest-ticket
 git+https://github.com/next-actions/pytest-tier
 git+https://github.com/next-actions/pytest-output
-git+https://github.com/SSSD/sssd-test-framework
+git+https://github.com/krishnavema/sssd-test-framework@smatcard
```

Comment:
Be careful with this.
Does this PR require another PR in sssd-test-framework metged first?

Reply:
Yes, it does, [in sssd-test-framework PR#179](https://github.com/SSSD/sssd-test-framework/pull/179).

And this entry in requirements.txt should be removed before this PR is merged.